### PR TITLE
Fix tags column name length. Set non-nullable user selection for post

### PR DESCRIPTION
**Pull request by DemianShtepa**
1. Tag name validation requires length to be less than 155, but the column length is 50.
2. It's allowed select empty user while creating post, which violates not null exception.

### CODE REVIEW

File: database/migrations/create_blog_tables.php.stub
Comment (AsmitNepali) [inline database/migrations/create_blog_tables.php.stub:1]: Please ensure you have to change on the application level not in the database and set the string length limit to 50. 155 does not seem optimal.

Comment (DemianShtepa) [reply database/migrations/create_blog_tables.php.stub:1]: @AsmitNepali Done.
Also changed slug max length which is also invalid.

Comment (AsmitNepali) [reply database/migrations/create_blog_tables.php.stub:1]: > @AsmitNepali Done. Also changed slug max length which is also invalid.

Thanks! for your contribution @DemianShtepa

**Review by AsmitNepali — APPROVED** [review]
Happy Contribution!

**Pull request merged**